python/hail/typecheck/check.py

```diff
@@ -115,7 +123,7 @@ def oneof(*args):
 
 
 def nullable(t):
-    return oneof(t, NoneType)
+    return oneof(t, type(None))
```

Comment:
Is only(type(None)) the same as type(None)? All the tests are passing so probably yes.